mne/decoding/time_gen.py

```diff
@@ -249,9 +290,25 @@ def predict(self, epochs):
         # Store all testing times parameters
         self.test_times_ = test_times
 
-        # Prepare parallel predictions across time points
+        n_orig_epochs, _, n_times = X.shape
```

Comment:
Because some sklearn cv are not complete (e.g. ShuffleSplit) meaning that some epochs do not get any prediction. Yet, if we want to vectorize the computations, we need contiguous arrays that we can slice, so we have some data prep and re-format at the beginning and at and the end.